entrypoint.sh

```diff
@@ -8,7 +8,11 @@ console_output() {
 # function to set custom parameters in UMS.conf
 set_ums_parameter() {
   console_output INFO "Setting UMS setting '${1}' to '${2}'"
-  sed -i "s+^${1} =$+${1} = ${2}+g" /opt/ums/UMS.conf
+  if $(grep -q "^${1} =" /opt/ums/UMS.conf); then
```

Comment:
Thanks for such a quick fix!  Only change that I would suggest would be to remove the `$()` as it's not needed. See [SC2091](https://github.com/koalaman/shellcheck/wiki/SC2091)

Reply:
Yep, fix done.